command/ls.go

```diff
@@ -239,6 +250,9 @@ const (
 
 // String returns the string representation of ListMessage.
 func (l ListMessage) String() string {
+	if l.showFullPath {
+		return l.Object.URL.String()
+	}
```

Comment:
I'm not sure that's what we want. I expected `ls` command output as-is, but are absolute instead of relative. With this flag, we only print full path without date and etag.